ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/ConnectorUtils.java

```diff
@@ -358,6 +360,32 @@ public static SdkHttpFullRequest buildSdkRequest(
         return builder.build();
     }
 
+    public static Request buildOKHttpRequestPOST(String action, Connector connector, Map<String, String> parameters, String payload) {
```

Comment:
it's specifically for streaming requests, not general POST requests. Can we rename the method accordingly?